<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceOnlineImportDialog</class>
 <widget class="QDialog" name="DeviceOnlineImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Blechuhr Data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="usernameComboBox"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QWebView" name="webView">
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="dateEdit">
        <property name="displayFormat">
         <string>MMMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="1">
          <widget class="QPushButton" name="mailPDFButton">
           <property name="text">
            <string>Mail as PDF</string>
           </property>
           <property name="icon">
            <iconset resource="../res/blechuhr.qrc">
             <normaloff>:/blechuhr/images/blechuhr/email.svg</normaloff>:/blechuhr/images/blechuhr/email.svg</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="printPDFButton">
           <property name="text">
            <string>Print to PDF</string>
           </property>
           <property name="icon">
            <iconset resource="../res/blechuhr.qrc">
             <normaloff>:/blechuhr/images/blechuhr/pdf_file.svg</normaloff>:/blechuhr/images/blechuhr/pdf_file.svg</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="exportCSVButton">
           <property name="text">
            <string>Export CSV</string>
           </property>
           <property name="icon">
            <iconset resource="../res/blechuhr.qrc">
             <normaloff>:/blechuhr/images/blechuhr/csv_file.svg</normaloff>:/blechuhr/images/blechuhr/csv_file.svg</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="4">
          <widget class="QListWidget" name="timesListWidget"/>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="mailCSVButton">
           <property name="text">
            <string>Mail as CSV</string>
           </property>
           <property name="icon">
            <iconset resource="../res/blechuhr.qrc">
             <normaloff>:/blechuhr/images/blechuhr/email.svg</normaloff>:/blechuhr/images/blechuhr/email.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>timesListWidget</zorder>
        <zorder>exportCSVButton</zorder>
        <zorder>printPDFButton</zorder>
        <zorder>mailPDFButton</zorder>
        <zorder>mailCSVButton</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/blechuhr.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeviceOnlineImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeviceOnlineImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
